cloud/logger/example/index.ts

```diff
@@ -1,20 +1,37 @@
 
-import {BunyanLogger, ConsoleLogger, Logger} from '../src/index';
+import { BunyanLogger, ConsoleLogger, getLogger, logger, Logger, setLogger} from '../src/index';
 
+let log: Logger = logger;
```

Comment:
[Antipattern] why we need to duplicate variable? 
[Antipattern] Better to use getter here?